api/authenticators/duo/duo.go

```diff
@@ -12,13 +13,21 @@ import (
 	"time"
 
 	"github.com/riotgames/key-conjurer/api/consts"
+)
 
-	"github.com/sirupsen/logrus"
+var (
+	ErrorDuoCommunication   = errors.New("unable to communicate with Duo")
+	ErrorDuoMfaNotAllow     = errors.New("MFA was not allowed")
+	ErrorDuoArgsError       = errors.New("there was an error parsing arguments for Duo push request")
+	ErrorDuoPushError       = errors.New("there was an error sending a Duo push request")
+	ErrorCannotFindSid      = errors.New("Cannot find sid")
+	ErrorHTTPBodyError      = errors.New("Unable to read http response body")
+	ErrorJSONMarshalError   = errors.New("Unable to marshal json")
+	ErrorJSONUnmarshalError = errors.New("Unable to unmarshal json")
 )
 
 // Duo scripts the Duo Web API interaction
 type Duo struct {
-	logger     *logrus.Entry
```

Comment:
The logging behind Duo was removed because all of the messages that *were* logged were always logged right before an error - so the messages were moved into the error handling so the caller can handle them.